src/main/java/com/github/lmiljak/dowertefence/commands/DTCommand.java

```diff
@@ -0,0 +1,25 @@
+package com.github.lmiljak.dowertefence.commands;
+
+import nl.taico.taeirlib.commands.CommandArgs;
+import nl.taico.taeirlib.commands.TCommand;
+
+/**
+ * Class for handling the /dt command and subcommands.
+ */
```

Comment:
You can, but you have to register it with the command framework. It's just like registering listeners, except for commands, and without the Listener interface.